src/time_evolution/brmesolve.jl

```diff
@@ -37,18 +37,16 @@ function bloch_redfield_tensor(
     U = QuantumObject(rst.vectors, Operator(), H.dimensions)
     sec_cutoff = float(sec_cutoff)
 
-    # in fock basis
-    R0 = liouvillian(H, c_ops)
+    H_new = getVal(fock_basis) ? H : QuantumObject(Diagonal(real.(rst.values)), Operator(), H.dimensions)
```

Comment:
```suggestion
    H_new = getVal(fock_basis) ? H : QuantumObject(Diagonal(rst.values), Operator(), H.dimensions)
```

I suggest we keep the imaginary parts of `Hamiltonian` eigenvalues. Just in case if someone wants to consider non-Hermitian Hamiltonian